content/rendering-with-batch/monitor/optional_fis.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Optional - Spot Interruption"
+date: 2022-09-20T00:00:00Z
+weight: 145
+---
+
+
+## This step is optional and is not required
+
+{{% notice warning %}}
+You should wait until the AWS Batch job from your first experiment has fully completed before starting this optional lab.
+{{% /notice %}}
+
+To experiment with how AWS Batch jobs handles EC2 Spot Interruptions, you can follow this guide...
+
+Start a rendering job, by initiating the state machine
+
+```
+export FIS_JOB_NAME="Pottery-FIS"
+export EXECUTION_ARN=$(aws stepfunctions start-execution --state-machine-arn "${StateMachineArn}" --input "{\"jobName\": \"${FIS_JOB_NAME}\", \"inputUri\": \"s3://${BucketName}/${BlendFileName}\", \"outputUri\": \"s3://${BucketName}/${FIS_JOB_NAME}\", \"jobDefinitionArn\": \"${JOB_DEFINITION_ARN}\", \"jobQueueArn\": \"${JOB_QUEUE_ARN}\", \"framesPerJob\": \"1\"}" | jq -r '.executionArn')
+echo "State machine started. Execution Arn: ${EXECUTION_ARN}."
+```
+
+Run the following command to generate the Fault Injection Simulator(FIS) Experiement template file. The FIS Experiemnt will create an interruption signal for some of the EC2 Spot instances.
```

Comment:
- AWS FIS
- Experiment should start with lower case
- It is not clear for which Spot Instances the interruption will be generated. What does the experiment do?